### DISCUSSION TITLE
running dkron.exe as windows service

### DISCUSSION THREAD

**Discussion by ihershkovitz** [dkron-io/dkron#1278] [Q&A] [answered]
How can I achieve running dkron.exe as windows service ?

**Comment by yvanoers** [comment, marked as answer]
Dkron wasn't written as a windows service, so by itself it won't run as a service on windows.
There are wrappers out there, such as [NSSM](https://nssm.cc/), that allow you to run a normal program as a service. I haven't tried it, though, YMMV.
